requirements-pymysql.txt

```diff
@@ -2,4 +2,5 @@ pytest
 coverage>=5
 pytest-cov
 flake8
+sqlglot
```

Comment:
is their model stable - need to pin version?

Reply:
they move versions very quickly, that being said, id like to always test with the latest, unless we bump into a reason not to  (these reqs are tests only)